docs/fundamentals/interface.rst

```diff
@@ -0,0 +1,105 @@
+
+Pipelines are executed by clicking the yellow play button in the bottom right of the workspace. In the window that appears, you can name the execution, select a computation platform, and select a storage platform. The computation platform can either be SciServer's Compute service or a WebGME platform. The available storage platforms are SciServer's Files service and Amazon's S3 service. The provided storage option will be used for storing both the output objects defined in the pipeline, as well as all files used in execution of the pipeline. Login credentials will be required for SciServer computation service, either storage service, and each individual input node in the pipeline.
+
+.. figure:: execute_pipeline.png
+    :align: center
+    :width: 75%
+
+Executions
+----------
+.. figure:: executions_tab.png
+    :align: center
+    :width: 75%
+
+This view allows the review of previous pipeline executions. Clicking on any execution will display any plotted data generated by the pipeline, and selecting multiple executions will display all of the selected plots together. Clicking the provided links will open either the associated pipeline or a trace of the execution (shown below). The blue icon in the top right of every node allows viewing the text output of that node. The execution trace can be viewed during execution to check the status of a running job. During execution, the color of a node indicates its current status. The possible statuses are:
+
+- **Dark gray**: Awaiting initialization
+- **Light gray**: Awaiting execution
+- **Yellow**: Currently executing
+- **Green**: Successfully finished execution
+- **Red**: Execution failed
+
+.. figure:: execution_finished.png
+    :align: center
+    :width: 50%
+
+Resources
+---------
+.. figure:: resources_tab.png
+    :align: center
+    :width: 75%
+
+This view shows the available neural network resources available to your pipelines. From this view, resources can be created, deleted, and renamed. Resources are arranged by the deepforge system and cannot by manually reordered.
+
+.. figure:: neural_network.png
+    :align: center
+    :width: 50%
+
+As with pipelines, the neural networks are depicted as directed graphs. Each node in the graph corresponds to a single layer or operation in the network (information on operations can be found on the `keras website <https://keras.io/api/>`_). Clicking on a node provides the ability to change the attributes of that layer, delete the layer, or add new layers before or after the current node. Many operations require that certain attributes be defined before use. The Conv2D node pictured above, for example, requires that the *filters* and *kernel_size* attributes be defined. If these are left as *<none>*, a visual indicator will show that there is an error to help prevent mistakes. In order to ease analysis and development, hovering over any connecting line will display the shape of the data as it moves between the given layers.
```

Comment:
"Clicking on a node" -> "Clicking on a layer"

Reply:
Changed in all but the first instance of the word "node".